sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -557,7 +565,7 @@ public virtual async Task StartProcessingAsync(CancellationToken cancellationTok
         ///
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> instance to signal the request to cancel the start operation.  This won't affect the <see cref="EventProcessorClient" /> once it starts running.</param>
         ///
-        /// <exception cref="EventHubsException">Occurs when this <see cref="EventProcessorClient" /> instance is already closed.</exception>
+        /// <exception cref="EventHubsException">Occurs when this <see cref="EventProcessorClient" /> instance is already closed; the <see cref="EventHubsException.FailureReason"/> will be set to <see cref="EventHubsException.FailureReason.ClientClosed"/>.</exception>
```

Comment:
Has the processor be changed to allow a "closed" state or is this piece of information simply inaccurate?

Reply:
Nope; this is just inaccurate and likely a remnant from when the processor was disposable/closable like the other clients.  I've trimmed the comment.